### ISSUE TITLE
onPress Card don't work after upgrade Expo to 51

### ISSUE THREAD

**Issue by deck2k** [callstack/react-native-paper#4450]
My code is 

  <Card style={[styles.buttonCard, styles.buttonColor]}
                                onPress={() => navigation.navigate('Property',
                                    {
                                      ---some properties--
                                    })}>
                             <View>--some text--</View>
<Card>

This code in expo 49 is functionally, after upgrade, card onPress is disabled and dont show content of view (icons, text)

If delete onPress , render view OK

Thanks!

**Comment by FouadMagdy01** [comment]
Dont make the Card component self-closed
Put some content inside and the onPress prop should work as expected 

This behavior also applies for the Button component 

I haven't explored the reason for this behavior in the component code but this is how it works

Hope it solve your problem

**Comment by deck2k** [comment]
excellent! There I modified the code and it worked perfectly.

Thank you very much for the quick answer!

**Comment by raajnadar** [comment]
@FouadMagdy01 do you mean to reproduce this bug don't send children?

**Comment by FouadMagdy01** [comment]
> @FouadMagdy01 do you mean to reproduce this bug don't send children?


Yes, When not providing the chuldren componentin React Native paper Components the pressing functionality doesn't work at all

You can even notice that if you have ```Button``` component the ripple effect on android will not trigger

**Comment by jb9229** [comment]
Menu.Item is also not working, onPress
